src/Backend/Core/Js/jquery/jquery.backend.js

```diff
@@ -1573,34 +1585,60 @@
       })
     },
 
+    buildNewString: function(oldString, fromBlockName, toBlockName, currentIndex, newIndex, prefix = '', suffix = '') {
+      var searchFor = fromBlockName
```

Comment:
I would add a check that you need to give a prefix or a postfix, just replacing a number doesn't seem very safe

Reply:
The prefix and suffix are optional, in most cases the prefix will be used, the suffix not.
Can you elaborate on what is not safe? In essence this is just a refactoring from the code before to have less similar code.

Reply:
I would add a check that checks that either a prefix or a postfix has been added.
If somebody tries to replace let's say 0 but it's also used on a different spot in the string that can give issues

Reply:
the fromBlockName is received from the Sortable event, and is something like `main`, `top`, and the currentIndex is appended, so it won't search for `0` only